examples/dynamic-bridge-app/linux/main.cpp

```diff
@@ -294,8 +294,12 @@ int main(int argc, char * argv[])
     clusterAccess.reserve(std::extent<decltype(clusters::kKnownClusters)>::value);
     for (auto & entry : clusters::kKnownClusters)
     {
-        clusterAccess.emplace_back(chip::Optional<EndpointId>(), entry.id);
-        registerAttributeAccessOverride(&clusterAccess.back());
+        // Desciptor clusters should not be overridden.
+        if (entry.id != 29)
```

Comment:
Please, no hard-coded numbers.

`zzz-generated/app-common/app-common/zap-generated/ids/Clusters.h` has `chip::app::Clusters::Descriptor::Id` that can be used here.